mycroft/client/enclosure/api.py

```diff
@@ -117,6 +117,29 @@ def eyes_color(self, r=255, g=255, b=255):
         self.ws.emit(Message("enclosure.eyes.color",
                              {'r': r, 'g': g, 'b': b}))
 
+    def eyes_setpixel(self, idx, r=255, g=255, b=255):
+        """Set individual pixels of the Mark 1 neopixel eyes
+        Args:
+            neopixel_idx (int): 0-11 for the right eye, 12-23 for the left
+            r (int): The red value to apply
+            g (int): The green value to apply
+            b (int): The blue value to apply
+        """
+        if idx < 0 or idx > 23:
+            return
```

Comment:
Would it make sense to `raise ValueError` here to let the programmer know they've made a mistake?